cfme/infrastructure/datastore.py

```diff
@@ -128,6 +129,19 @@ def is_displayed(self):
         )
 
 
+class DatastoreManagedVMsView(VmsTemplatesAllView):
+    """
+    This view is used in test_datastore_relationships
```

Comment:
**Required**
I think this could be something like `represents All VMs and Templates page for datastores`